cfssl/README.md

```diff
@@ -0,0 +1,51 @@
+# Agent Check: cfssl
+
+## Overview
+
+This check monitors [cfssl][1] through the Datadog Agent.
+
+## Setup
+
+Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][3] for guidance on applying these instructions.
+
+### Installation
+
+The cfssl check is not included in the [Datadog Agent][2] package, so you need to install it.
```

Comment:
Could you confirm if this is still accurate? It doesn't seem that anything is being installed

Reply:
Updated based on another integration